e2e/e2etests/test_admin_transactions.go

```diff
@@ -0,0 +1,103 @@
+package e2etests
+
+import (
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/zetacore/e2e/runner"
+	"github.com/zeta-chain/zetacore/e2e/utils"
+	"github.com/zeta-chain/zetacore/pkg/chains"
+	"github.com/zeta-chain/zetacore/pkg/coin"
+	"github.com/zeta-chain/zetacore/testutil/sample"
+	crosschaintypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+// TestCriticalAdminTransactions tests critical admin transactions that are the most used on mainnet .
+// The complete list is
+// MsgUpdateChainParams
+// MsgRefundAbortedCCTX
+// MsgEnableCCTX
+// MsgDisableCCTX
+// MsgUpdateGasPriceIncreaseFlags
+// MsgAddInboundTracker
+// MsgUpdateZRC20LiquidityCap
+// MsgDeploySystemContracts
+// MsgWhitelistERC20
+// MsgPauseZRC20
+// MsgMigrateTssFunds
+// MsgUpdateTssAddress
+//
+//	However, the transactions other than `AddToInboundTracker` and `UpdateGasPriceIncreaseFlags` have already been used in other tests.
+func TestCriticalAdminTransactions(r *runner.E2ERunner, _ []string) {
+	TestAddToInboundTracker(r)
+	TestUpdateGasPriceIncreaseFlags(r)
+}
+
+func TestUpdateGasPriceIncreaseFlags(r *runner.E2ERunner) {
+	defaultFlags := observertypes.DefaultGasPriceIncreaseFlags
```

Comment:
Can you please add comments to this test case? Also you can use "arrange", "act", "assert" style. It really helps to grasp the context of the test